### ISSUE TITLE
[CI] XPack full-cluster-restart qa test unrecognized VM option error

### ISSUE THREAD

**Issue by martijnvg** [elastic/elasticsearch#30167]
Unrecognized VM option 'UseParNewGC':

```
06:38:19 Starting process 'command 'cmd''. Working directory: C:\Users\jenkins\workspace\elastic+elasticsearch+6.3+multijob-windows-compatibility\x-pack\qa\full-cluster-restart\with-system-key\build\cluster\v5.6.10-SNAPSHOT#oldClusterTestCluster node0\cwd Command: cmd /C call C:\Users\jenkins\WORKSP~1\ELASTI~1.3_M\x-pack\qa\FULL-C~1\WITH-S~1\build\cluster\V56~1.10-\ELASTI~1.10-\bin\x-pack\users useradd test_user -p x-pack-test-password -r superuser
06:38:19 Successfully started process 'command 'cmd''
06:38:19 Unrecognized VM option 'UseParNewGC'
06:38:19 Error: Could not create the Java Virtual Machine.
06:38:19 Error: A fatal exception has occurred. Program will exit.
06:38:19 :x-pack:qa:full-cluster-restart:with-system-key:v5.6.10-SNAPSHOT#oldClusterTestCluster#node0.setupTestUser FAILED
06:38:19 :x-pack:qa:full-cluster-restart:with-system-key:v5.6.10-SNAPSHOT#oldClusterTestCluster#node0.setupTestUser (Thread[Daemon worker,5,main]) completed. Took 0.044 secs.
06:38:19 
06:38:19 FAILURE: Build failed with an exception.
06:38:19 
06:38:19 * What went wrong:
06:38:19 Execution failed for task ':x-pack:qa:full-cluster-restart:with-system-key:v5.6.10-SNAPSHOT#oldClusterTestCluster#node0.setupTestUser'.
06:38:19 > Process 'command 'cmd'' finished with non-zero exit value 1
```

Build url: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.3+multijob-windows-compatibility/4/console

**Comment by martijnvg** [comment]
Note the vm option is unrecognised, because the build used Java 10.

**Comment by jasontedor** [comment]
This is fixed already by 150a53b6085ff9ef13d07e5113a316dfe7e62d81 but this build was on an ancestor of that commit: 72ade04

Relates #29647

**Comment by martijnvg** [comment]
@jasontedor Thanks. I assumed the build had this change, I didn't look close enough what commit this build used.
